doc/source/config.rst

```diff
@@ -0,0 +1,130 @@
+Configuration
+=============
+
+Satpy has two levels of configuration that allow to control how Satpy and
+its various components behave. There are a series of "settings" that change
+the global Satpy behavior. There are also a series of "component
+configuration" YAML files for controlling the complex functionality in readers,
+compositors, writers, and other Satpy components that can't be controlled
+with traditional keyword arguments.
+
+Settings
+--------
+
+There are configuration parameters in Satpy that are not specific to one
+component and control more global behavior of Satpy. These parameters can be
+set in one of three ways:
+
+1. Environment variable
+2. YAML file
+3. At runtime with ``satpy.config``
+
+This functionality is provided by the :doc:`donfig <donfig:configuration>`
+library. The currently available settings are described below.
+Each option is available from all three methods. If specified as an
+environment variable or specified in the YAML file on disk, it must be set
+**before** Satpy is imported.
+
+**YAML Configuration**
+
+YAML files that include these parameters can be in any of the following
```

Comment:
A short example of the satpy.yaml config file would be nice here.

Reply:
Added.